keras_nlp/models/roberta/roberta_models.py

```diff
@@ -86,12 +85,10 @@ def __init__(
         intermediate_dim,
         dropout=0.1,
         max_sequence_length=512,
-        name=None,
-        trainable=True,
+        **kwargs,
     ):
+        start_token_index = 0
```

Comment:
You could remove this line and define this constant directly in line 154 since it's not used in the encoder directly.